src/Sylius/Behat/Context/Api/Shop/CustomerContext.php

```diff
@@ -160,6 +162,21 @@ public function iChangePasswordTo(string $oldPassword, string $newPassword): voi
         ]);
     }
 
+    /**
+     * @When /^(I) try to verify my account using the link from this email$/
+     */
+    public function iTryToVerifyMyAccountUsingTheLinkFromEmail(ShopUserInterface $user): void
+    {
+        $request = Request::custom(
+            \sprintf('/api/v2/shop/customers/%s/account-verify', (string) $user->getId()),
```

Comment:
```suggestion
            \sprintf('/api/v2/shop/customers/%s/verify-account', (string) $user->getId()),
```
to be more consistent with current routes like for example `change-password` or `apply-coupon`